components/binary_sensor/index.rst

```diff
@@ -60,6 +60,10 @@ Advanced options:
 - **disabled_by_default** (*Optional*, boolean): If true, then this entity should not be added to any client's frontend,
   (usually Home Assistant) without the user manually enabling it (via the Home Assistant UI).
   Requires Home Assistant 2021.9 or newer. Defaults to ``false``.
+- **entity_category** (*Optional*, string): The category of the entity.
+  See https://developers.home-assistant.io/docs/core/entity/#generic-properties.
```

Comment:
```suggestion
  See https://developers.home-assistant.io/docs/core/entity/#generic-properties
```
Same everywhere below.

Reply:
I guess I added the dot after testing one..haha